Dockerfile

```diff
@@ -356,6 +369,21 @@ RUN cd libs && source ./install_JSAnimation.sh
 # http://ipyparallel.readthedocs.org/en/latest/
 RUN ipcluster nbextension enable
 
+#Massia: Integracao com o LDAP via SSSD
+RUN yum -y install \
+        sssd \
+        && yum clean all
+
+RUN /usr/bin/chmod 600 /etc/sssd/sssd.conf
+
+#Massia: arquivo PAM de login do RStudio
+ADD rootfs/etc/pam.d/rstudio /etc/pam.d/rstudio
```

Comment:
Vou adicionar amanhã!

On Sunday, 19 June 2016, Felipe Noronha <notifications@github.com> wrote:

> In Dockerfile
> <https://github.com/felipenoris/math-server-docker/pull/6#discussion_r67625767>
> :
>
> > @@ -356,6 +369,21 @@ RUN cd libs && source ./install_JSAnimation.sh
> >  # http://ipyparallel.readthedocs.org/en/latest/
> >  RUN ipcluster nbextension enable
> >
> > +#Massia: Integracao com o LDAP via SSSD
> > +RUN yum -y install \
> > +        sssd \
> > +        && yum clean all
> > +
> > +RUN /usr/bin/chmod 600 /etc/sssd/sssd.conf
> > +
> > +#Massia: arquivo PAM de login do RStudio
> > +ADD rootfs/etc/pam.d/rstudio /etc/pam.d/rstudio
>
> também não achou este arquivo.
>
> Step 55 : ADD rootfs/etc/pam.d/rstudio /etc/pam.d/rstudio
> lstat rootfs/etc/pam.d/rstudio: no such file or directory
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/felipenoris/math-server-docker/pull/6/files/e8c847c640775ad266ac682b38c3d424ff99530c#r67625767>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ALAy9KQZTZj8gAnwhThDfQb_E5KD5smRks5qNdBSgaJpZM4I4u1M>
> .
>